### ISSUE TITLE
Space between value and unit of measure

### ISSUE THREAD

**Issue by xsirpreisx** [flixlix/power-flow-card-plus#36]
Since the latest version of the card the unit of measure (W/kW) is written right after the value (e.g. 10W / 20kW). Me nitpicking again, this is "wrong". There is "supposed to be" a space between the number and the unit (10 W / 20 kW / 30 kWh). 

The space/blank was removed here: https://github.com/flixlix/power-flow-card-plus/commit/42e609d4ab024c51efaf6a4e765224b7c8dd83e4

We should either change the code back to always use a space between the two, or give the user an option to choose.

No blank/space gives more room, but it always kind of looks wrong to my eye.

What do you think?

**Comment by flixlix** [comment]
No, I agree with you that what makes the whole experience lies in the details, so let's correct that. 🦾
That was in fact my mistake, I wanted to only change the white space for the `secondary_info` in the individual devices, to match the battery. For now I'll revert this change and maybe later I'll add an option to remove the white space again, give me a few minutes

**Comment by flixlix** [comment]
Hopefully fully fixed 🤞 https://github.com/flixlix/power-flow-card-plus/releases/tag/v0.0.9.5

**Comment by xsirpreisx** [comment]
You're very fast! 🚀 

Much better, thank you!

I would prefer a white space for battery SoC as well. ;-)

What about secondary_info for solar? Are you planing on implementing that?

**Comment by flixlix** [comment]
Thanks, I'm trying 😅
I'll add a `unit_white_space` for the battery.

I have thought of `secondary_info` for non fossil, home and solar. Currently I'm working on some other, cooler features 😉

**Comment by xsirpreisx** [comment]
Looking forward to them!
